src/OpenTelemetry.Extensions/README.md

```diff
@@ -84,3 +84,27 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
 Warning: The baggage key predicate is executed for every baggage entry for each
 started activity.
 Do not use slow or intensive operations.
+
+### RateLimitingSampler
+
+The rate limiting sampler is a sampler that will limit the number of traces to
+the specified rate per second. It is typically used in conjunction with the ParentBasedSampler
+to ensure that the rate limiting sampler is only applied to the root spans. If
+a request comes in without a sampling decision, the rate limiting sampler will
+make a decision based on the rate limit. The sampling decision is stored in the
```

Comment:
```suggestion
make a decision based on the rate limit.
```

The last sentence is not really required here. That is something OTel SDK does, irrespective of the Sampler